src/controller/python/chip/clusters/Attribute.py

```diff
@@ -791,12 +791,12 @@ def _handleReportEnd(self):
 
         if (self._subscription_handler is not None):
             for change in self._changedPathSet:
+                attribute_path = change
                 try:
                     attribute_path = TypedAttributePath(Path=change)
-                except (KeyError, ValueError) as err:
+                except KeyError as err:
                     # path could not be resolved into a TypedAttributePath
-                    logging.getLogger(__name__).exception(err)
-                    continue
+                    logging.getLogger(__name__).warning(err)
```

Comment:
I'd even drop this down to debug level as its handled now. Maybe even adjust the error message to something nicer than just the exception?